recipes/icu/all/conanfile.py

```diff
@@ -234,16 +234,28 @@ def _data_filename(self):
     def package_info(self):
         self.cpp_info.names["cmake_find_package"] = "ICU"
         self.cpp_info.names["cmake_find_package_multi"] = "ICU"
+
         # icudata
         self.cpp_info.components["icu-data"].names["cmake_find_package"] = "data"
         self.cpp_info.components["icu-data"].names["cmake_find_package_multi"] = "data"
         self.cpp_info.components["icu-data"].libs = [self._lib_name("icudt" if self.settings.os == "Windows" else "icudata")]
         if not self.options.shared:
             self.cpp_info.components["icu-data"].defines.append("U_STATIC_IMPLEMENTATION")
+
+        # icu uses c++, so add the c++ runtime
+        libcxx = self.settings.get_safe("compiler.libcxx")
```

Comment:
why not tools.cppstd_library?

Reply:
I know of [`tools.cppstd_flag`](https://docs.conan.io/en/latest/reference/tools.html#tools-cppstd-flag), but not `tools.cppstd_library`.

Reply:
Ok, you meant [tools.stdcpp_library](https://docs.conan.io/en/latest/reference/tools.html#tools-stdcpp-library)